src/core/lib/support/time_posix.cc

```diff
@@ -42,7 +42,7 @@ static struct timespec timespec_from_gpr(gpr_timespec gts) {
   return rv;
 }
 
-#if _POSIX_TIMERS > 0
+#if _POSIX_TIMERS > 0 || defined(__OpenBSD__)
```

Comment:
Doesn't OpenBSD set `_POSIX_TIMERS` in its `unistd.h`?

Reply:
It sets it as `-1`.